src/server/server.js

```diff
@@ -32,42 +37,25 @@ mongoose
   )
   .catch((err) => log(serverLog(err)));
 
-// **** middleware ****//
```

Comment:
Tell me why we deleted this?

Reply:
I was waging a war against the green text 👊